### DISCUSSION TITLE
Version 7.x on Win7

### DISCUSSION THREAD

**Discussion by ivanbuto** [RandomEngy/VidCoder#974] [General]
Will versions 7.x of VidCoder based on HandBrake 1.5 not work at all on Windows 7 (or 8.1)?
Version 6.45 runs fine on my Windows 7 x64 machine with relatively recent hardware.
Microsoft .NET Desktop Runtime 6 is still supported on Windows 7.
Not a criticism, just a question. :)
Thank you.

**Comment by RandomEngy** [comment]
It may or may not work. None of the developers will test on it and if you have any trouble, you will be on your own. I have not added any code to explicitly block installation on Windows 7 though.

**Reply by ivanbuto** [reply]
Thanks for the information.
I get an error when trying to install, as follows:
![image](https://user-images.githubusercontent.com/34988265/153951812-c9ae2373-fc69-4a87-957c-81a47ab74be8.png)
Probably because I don't have .NET Desktop Runtime 6 installed. Will install that separately and give it a go again.

**Reply by ivanbuto** [reply]
So installing .NET Desktop Runtime 6.0.2 didn't help, this dll file is not contained there.
Guess it is bundled with some other component that is Windows 10 specific.

**Reply by ivanbuto** [reply]
Lastly, installed also the latest Visual C++ 2015-2022 Redistributable package, which did not help either.
Found this resource though:
https://github.com/dotnet/core/issues/6976
Anyway, sorry to bother you.

**Reply by Darthagnon** [reply]
Have you tried [this api ms win core DLL modification](https://github.com/nalexandru/api-ms-win-core-path-HACK)? It has been reported to make Python 3.9 and higher, as well as recent Blender, work on Win7

**Reply by ivanbuto** [reply]
Tried that, but it didn't help. Probably because the missing file is "api-ms-win-core-**winrt**-l1-1-0.dll" rather than "api-ms-win-core-**path**-l1-1-0.dll".

**Reply by ivanbuto** [reply]
So I ended up, with some reluctance, downloading both 32-bit and 64-bit versions of "api-ms-win-core-winrt-l1-1-0.dll from dll-files.com and copying them in the %windir%\SysWOW64 and %windir%\System32 directories respectively. After this, I installed Vidcoder 7.12 successfully!
